example-integrations/weaviate/weaviate-semantic-search.py

```diff
@@ -1,11 +1,3 @@
-# PREREQUISITE:
```

Comment:
Are these no longer pertinent?

Reply:
I moved them into the README last PR but forgot to remove this here